src/main/java/org/example/daos/SalesEmployeeDao.java

```diff
@@ -0,0 +1,81 @@
+package org.example.daos;
+
+import org.example.models.SalesEmployee;
+import org.example.models.SalesEmployee;
+import org.example.models.SalesRequest;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+
+public class SalesEmployeeDao {
```

Comment:
Looks like we've story 1 and 4 in the one PR. One of the non-functionals is that it's code for one story only. Please separate